### ISSUE TITLE
installing postgresql twice

### ISSUE THREAD

**Issue by ltalirz** [aiidalab/aiidalab-docker-stack#23]
@yakutovicha You're currently installing postgresql twice in the aiidalab-1.0-migration branch:

version 10
https://github.com/aiidalab/aiidalab-docker-stack/blob/530c7f5531a4bc76fb40c51ef66dfe632e18ad80/Dockerfile#L41

and version 9.6
https://github.com/aiidalab/aiidalab-docker-stack/blob/530c7f5531a4bc76fb40c51ef66dfe632e18ad80/Dockerfile#L72-L77

What ends up running is version 10:
```
# ps -ef | grep postgresql
scienti+    43     1  0 19:59 ?        00:00:00 /usr/lib/postgresql/10/bin/postgres -D /project/.postgresql
```

This will bite you when you try to migrate databases from the previous image that was running postgres 9.6.
Either you add some [migration scripts](https://www.postgresql.org/docs/10/upgrading.html) or you switch back to version 9.6

**Comment by yakutovicha** [comment]
I choose to run version 10 and provide migration scripts. This is anyways done for aiida-1.0 (0.12 uses Postgres 9.6)

Thanks for catching this error!

**Comment by yakutovicha** [comment]
This is now fixed, I created a separate issue #25 to remind myself about migration scripts.
